docs/reference/releases/0.12.0.cue

```diff
@@ -0,0 +1,211 @@
+package metadata
+
+releases: "0.12.0": {
+	date:     "2021-02-11"
+	codename: "Vector Remap Language"
+	description: """
+		The Vector team is pleased to announce 0.12.0. This release	introduces the **Vector Remap Language**, an
```

Comment:
There are a few accidental tabs on this page. Not sure if that matters in the presented text.